### PR TITLE
Clarification around native build

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 1]
```diff
@@ -201,6 +201,7 @@ mvn package -Pnative -Dnative-image.docker-build=true
 
 The produced executable will be a 64 bit Linux executable, so depending on your operating system it may no longer be runnable.
 However, it's not an issue as we are going to copy it to a Docker container.
+Note that in this case the build itself runs in a Docker container, too, so you don't need to have GraalVM installed locally.
```
Comment (geoand) [inline docs/src/main/asciidoc/maven-tooling.adoc]: `Docker container, too` perhaps should be `Docker container too`?

Comment (Sanne) [reply docs/src/main/asciidoc/maven-tooling.adoc]: I think so, too :P

Comment (geoand) [reply docs/src/main/asciidoc/maven-tooling.adoc]: Pun intended I guess :P

**Review by gsmet — CHANGES_REQUESTED** [review]
@gunnarmorling I let you fix the comma? Thanks.

File: docs/src/main/asciidoc/maven-tooling.adoc (hunk 1 above)
Comment (gunnarmorling) [reply docs/src/main/asciidoc/maven-tooling.adoc]: Ah, those subtleties always so tough for us non-native speakers :) Just [double checked](https://www.grammarly.com/blog/comma-before-too/), the comma there *may* be added. I'll remove it, though, as there seems common dislike of it :)

Comment (gunnarmorling) [reply docs/src/main/asciidoc/maven-tooling.adoc]: Comma removed.

**Comment by gunnarmorling** [comment]
Done.

**Review by gsmet — APPROVED** [review]

File: docs/src/main/asciidoc/maven-tooling.adoc (hunk 1 above)
Comment (cescoffier) [reply docs/src/main/asciidoc/maven-tooling.adoc]: Are we sure about this sentence? I believe the plugin check for GRAALVM_HOME. (`QuarkusNative:213`)

Comment (gunnarmorling) [reply docs/src/main/asciidoc/maven-tooling.adoc]: Seems like that code doesn't run in that case; at least I don't have GraalVM installed and can build a container image with a native binary via `mvn clean install -Pnative -Dnative-image.docker-build=true`. Wheres an error is raised when omitting the -D property:

> Failed to generate a native image: GRAALVM_HOME was not set

Comment (cescoffier) [reply docs/src/main/asciidoc/maven-tooling.adoc]: Great! So there is a switch somewhere ;-)

**Pull request merged**